### ISSUE TITLE
Keep numeric order of rules

### ISSUE THREAD

**Issue by stefonarch** [WayfireWM/wf-config#60]
Actual: Using `wcm` windows rules are sorted by first digit only:
![screen_area_sab_18:12:36_](https://user-images.githubusercontent.com/10681413/222919567-0c203d2b-b0a4-4aef-9c3e-1509e2c51aba.png)

Expected:

rules are in numeric order also if >9

**Comment by marcusbritanicus** [comment]
@stefonarch FYI: the `rule_` prefix is not needed. it might be easier for you to sort them according to your app. For example, `panel_N` for rules regarding the panel or `desktop_M` for desktop. You don't even need to specify a number. You can have something abstract like `panel_bottom_sticky` or `make_pcmanfm_background` as rule names.

**Comment by stefonarch** [comment]
Yeah, just learned that, maybe this could be a comment in `wayfire.ini`, I took for granted the syntax - 
and didn't like it much ;)

**Comment by marcusbritanicus** [comment]
I'll update the wiki... :)
